docs/public-networks/how-to/upgrade-node.md

```diff
@@ -6,28 +6,171 @@ tags:
   - public networks
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Upgrade your Besu node
 
 When upgrading your Besu node, we recommend:
 
-- Using an orchestration method (for example, Ansible or Chef) to keep all nodes in sync with your desired configuration.
+- Preserving your node's data and configuration
+<!--- - Using an orchestration method  to keep all nodes in sync with your desired configuration. --->  
 - Storing your configuration under version control.
 
-## Ansible
 
-You can use the [Ansible role on Galaxy](https://galaxy.ansible.com/consensys/hyperledger_besu) directly or customize it to suit your needs.
+<Tabs>
+  <TabItem value="binary" label="Linux" default>
+
+### Upgrading on Linux
+
+1. Stop your Besu node:
+
+```bash
+sudo systemctl stop besu
+```
+
+2. Back up your data directory and configuration:
+
+```bash
+sudo cp -r /var/lib/besu/data /var/lib/besu/data.backup
+sudo cp /etc/besu/config.toml /etc/besu/config.toml.backup
+```
+
+3. Download and install the new version:
```

Comment:
The coincashew script that uses https://api.github.com/repos/hyperledger/besu/releases/latest is good.
Think we should credit them if we copy it though.